Core/Utilities/Plotting/ObjectLoader.py

```diff
@@ -0,0 +1,51 @@
+import re
+import os
+import DIRAC
+from DIRAC                                       import gLogger
+from DIRAC.Core.Utilities                        import List
+from DIRAC.ConfigurationSystem.Client.Helpers    import CSGlobals
+
+def loadObjects( path, reFilter = None, parentClass = None ):
```

Comment:
Without looking at the details, there is already an object loader for the Request operations, and for the plugin of the conditionnal FC instanciation, and probably for many other things. Can't we group all that ?